### PR TITLE
Fix broken links for community build

### PR DESCRIPTION
**Pull request by mukul-seagate11**
Signed-off-by: Mukul Malhotra <mukul.malhotra@seagate.com>

### Describe your changes in brief
Fix broken links for community build which caused due to folder restructure

### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [x] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [x] tested locally
 - [ ] added new dependencies
 - [x] updated the docs
 - [ ] added a test

-----
[View rendered doc/community-build/cloud/AWS/README.md](https://github.com/Seagate/cortx/blob/mukul-communitybuild-patch1/doc/community-build/cloud/AWS/README.md)

### CODE REVIEW

File: doc/community-build/cloud/AWS/README.md [hunk 1]
```diff
@@ -77,19 +77,19 @@ This file consists of the procedure to compile complete CORTX stack and Deploy o
...
+   After CORTX build is ready follow [CORTX Deployment](https://github.com/Seagate/cortx/blob/main/doc/community-build/archive/ProvisionReleaseBuild.md) to deploy CORTX on AWS instance. Please exclude SELINUX and Hostname setup steps.
 
 ### 3. OnBoarding 
 
    Configure CORTX by following [Onboarding](https://github.com/Seagate/cortx/blob/main/doc/Preboarding_and_Onboarding.rst) steps.
 
 ### 4. Testing IO operations 
 
-   We will need separate AWS instance to test IO operations. Follow [IO Test using AWS ](https://github.com/Seagate/cortx/blob/main/doc/integrations/AWS_EC2.md#step-7-create-another-ec2-instance-to-access-to-act-as-the-s3-client) and subsequent steps. 
+   We will need separate AWS instance to test IO operations. Follow [IO Test using AWS ](https://github.com/Seagate/cortx/blob/main/doc/integrations/AWS_EC2/README.md) and subsequent steps.
```
Comment (gauravchaudhari02) [inline doc/community-build/cloud/AWS/README.md:92]: @mukul-seagate11, looks like it pointing to cortx setup using ec2 and not IO using AWS. do we have another link to direct test point?

Comment (mukul-seagate11) [reply doc/community-build/cloud/AWS/README.md:92]: @gauravchaudhari02, it the same link for both where IO section is included

Comment (mukul-seagate11) [reply doc/community-build/cloud/AWS/README.md:92]: No, other then this unable to find the IO refs using AWS

**Review by hessio — COMMENTED** [review]
Looks good.

**Review by hessio — APPROVED** [review]
Looks good to me

File: doc/community-build/cloud/AWS/README.md [hunk 2]
```diff
@@ -77,19 +77,19 @@ This file consists of the procedure to compile complete CORTX stack and Deploy o
 
 ### 1. CORTX Build
 
-   We will use [cortx-build](https://github.com/Seagate/cortx/pkgs/container/cortx-build) docker image to compile entire CORTX stack. Please follow [CORTX compilation](https://github.com/Seagate/cortx/blob/main/doc/community-build/Generate-Cortx-Build-Stack.md) steps for compilation.
+   We will use [cortx-build](https://github.com/Seagate/cortx/pkgs/container/cortx-build) docker image to compile entire CORTX stack. Please follow [CORTX compilation](https://github.com/Seagate/cortx/blob/main/doc/community-build/archive/Generate-Cortx-Build-Stack.md) steps for compilation.
```
Comment (shailesh-vaidya) [inline doc/community-build/cloud/AWS/README.md:80]: CORTX Compilation should link to https://github.com/Seagate/cortx/blob/main/doc/community-build/docker/cortx-all/README.md

File: doc/community-build/cloud/AWS/README.md [hunk 3]
```diff
@@ -77,19 +77,19 @@ This file consists of the procedure to compile complete CORTX stack and Deploy o
...
 
 ### 2. CORTX Deployment
 
-   After CORTX build is ready follow [CORTX Deployment](https://github.com/Seagate/cortx/blob/main/doc/community-build/ProvisionReleaseBuild.md) to deploy CORTX on AWS instance. Please exclude SELINUX and Hostname setup steps.
+   After CORTX build is ready follow [CORTX Deployment](https://github.com/Seagate/cortx/blob/main/doc/community-build/archive/ProvisionReleaseBuild.md) to deploy CORTX on AWS instance. Please exclude SELINUX and Hostname setup steps.
```
Comment (shailesh-vaidya) [inline doc/community-build/cloud/AWS/README.md:84]: CORTX Deployment should link to https://github.com/Seagate/cortx-k8s/blob/main/doc/cortx-aws-k8s-installation.md

File: doc/community-build/cloud/AWS/README.md [hunk 4]
```diff
@@ -77,19 +77,19 @@ This file consists of the procedure to compile complete CORTX stack and Deploy o
...
 
 ### 3. OnBoarding
```
Comment (shailesh-vaidya) [inline doc/community-build/cloud/AWS/README.md:86]: This section should be removed

File: doc/community-build/cloud/AWS/README.md [hunk 5]
```diff
@@ -77,19 +77,19 @@ This file consists of the procedure to compile complete CORTX stack and Deploy o
...
 ### 2. CORTX Deployment
 
-   After CORTX build is ready follow [CORTX Deployment](https://github.com/Seagate/cortx/blob/main/doc/community-build/ProvisionReleaseBuild.md) to deploy CORTX on AWS instance. Please exclude SELINUX and Hostname setup steps.
+   After CORTX build is ready follow [CORTX Deployment](https://github.com/Seagate/cortx/blob/main/doc/community-build/archive/ProvisionReleaseBuild.md) to deploy CORTX on AWS instance. Please exclude SELINUX and Hostname setup steps.
 
 ### 3. OnBoarding 
 
    Configure CORTX by following [Onboarding](https://github.com/Seagate/cortx/blob/main/doc/Preboarding_and_Onboarding.rst) steps.
 
 ### 4. Testing IO operations
```
Comment (shailesh-vaidya) [inline doc/community-build/cloud/AWS/README.md:90]: This section should be removed as its an obsolete.

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

File: doc/community-build/cloud/AWS/README.md (hunk 3 above)
Comment (mukul-seagate11) [reply doc/community-build/cloud/AWS/README.md:84]: Done

File: doc/community-build/cloud/AWS/README.md (hunk 2 above)
Comment (mukul-seagate11) [reply doc/community-build/cloud/AWS/README.md:80]: Done

File: doc/community-build/cloud/AWS/README.md (hunk 4 above)
Comment (mukul-seagate11) [reply doc/community-build/cloud/AWS/README.md:86]: Done

File: doc/community-build/cloud/AWS/README.md (hunk 5 above)
Comment (mukul-seagate11) [reply doc/community-build/cloud/AWS/README.md:90]: Done

**Review by shailesh-vaidya — APPROVED** [review]

**Pull request merged**